Tutorials/Reduction/README.md

```diff
@@ -0,0 +1,25 @@
+# Reduction Case Study
+
+This repository hosts the sample code used in the HIP documentation.
+
+## Structure
+
+The code structure follows mostly that of
+[rocPRIM](https://github.com/ROCmSoftwarePlatform/rocPRIM), differing in a few
```

Comment:
differing in a few ways:

Reply:
"differing in a few ways:" fixed

I tried to rephase it. Can you take a look at it, please?